### PR TITLE
Created map with markers from dummy data

### LINKED ISSUE #17: Create map interface using google maps
As a user I would like to see a clear responsive map that can allow me to search for cities. 

Acceptance Criteria
- [ ] Map with markers that show available property.

### PR DESCRIPTION
**Pull request by hoslack**
Closes #17

### CODE REVIEW

File: client/src/components/App.js [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
 class App extends Component {
-  render() {
-    return (
-      <div className="container">
-        <Header />
-        <SearchForm />
-      </div>
-    );
-  }
+	render() {
```
Comment (claireinez) [inline client/src/components/App.js]: You've used hard tabs here (you can tell as GitHub renders them as 8 spaces). Please be consistent with tabs or spaces as you've used spaces everywhere else.

File: client/src/components/MapWithMarker.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+import { withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
+
+const MapWithAMarker = withGoogleMap(props => (
```
Comment (claireinez) [inline client/src/components/MapWithMarker.js]: The file is called `MapWithMarker` so this component should be named the same.

File: client/src/components/MapWithMarker.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
```
Comment (claireinez) [inline client/src/components/MapWithMarker.js]: `Component` isn't used in this file so it doesn't need to be imported.

File: client/src/components/MapContainer.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import MapWithMarker from './MapWithMarker';
+
+class MapContainer extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			markers: [
+				{ lat: 31.705791, lng: 35.200657 },
```
Comment (claireinez) [inline client/src/components/MapContainer.js]: I thought you were planning to have this data come from a dummy API response JSON file?

File: client/src/components/MapContainer.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+				{ lat: 31.771959, lng: 35.217018 },
+			],
+		};
+	}
+
+	render() {
+		return (
+			<div>
+				<MapWithMarker
+					containerElement={<div style={{ height: `400px` }} />}
```
Comment (claireinez) [inline client/src/components/MapContainer.js]: Why are you using backticks instead of quote marks?

File: client/src/components/MapContainer.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+					mapElement={<div style={{ height: `100%` }} />}
```
Comment (claireinez) [inline client/src/components/MapContainer.js]: Why are you using backticks instead of quote marks?

File: client/src/components/MapWithMarker.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+import { withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
+
+const MapWithAMarker = withGoogleMap(props => (
+	<GoogleMap defaultZoom={8} defaultCenter={{ lat: 32.699635, lng: 35.303547 }}>
```
Comment (claireinez) [inline client/src/components/MapWithMarker.js]: How did you decide this defaultCenter lat and long? I think this should be declared as a `const defaultCenter` at the top of the file with a comment stating where this location is.

**Review by claireinez — CHANGES_REQUESTED** [review]
Please change the hard tabs to spaces! It makes it really difficult to read on GitHub like this!

File: client/src/components/App.js (hunk 1 above)
Comment (hoslack) [reply client/src/components/App.js]: We use prettier, which does the automatic indentation. I don't get to decide between tabs and spaces @claireinez

File: client/src/components/MapContainer.js (hunk 4 above)
Comment (hoslack) [reply client/src/components/MapContainer.js]: That is what we are planning to do in the next issue.

File: client/src/components/MapWithMarker.js (hunk 2 above)
Comment (hoslack) [reply client/src/components/MapWithMarker.js]: We have renamed it to the name of the file.

**Review by claireinez — APPROVED** [review]

**Pull request merged**